README.md

```diff
@@ -1,2 +1,6 @@
 # http-server
 Http server built for python 401 (week 2)
+Created by Kyle Richardson and Patrick Trompeter.
+
+# As of 3/8/16
+This server returns a HTTP OK response when a client sends some kind of request. The server also prints to its local stdout the incoming request.  
```

Comment:
I would be good to make this much more descriptive regarding what your server is actually doing. Hopefully that can happen by the time your final server is built.